### PR TITLE
Add AWS tower config

### PR DESCRIPTION
**Pull request by ggabernet**
Adding AWS tower profile config, needed for the automated AWS megatests.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     needs: test_all_profiles
     strategy:
         matrix:
-          profile: ['abims', 'awsbatch', 'bi','bigpurple', 'binac', 'biohpc_gen', 'cbe', 'ccga_dx', 'ccga_med', 'cfc', 'cfc_dev', 'crick', 'denbi_qbic', 'ebc', 'eddie', 'eva', 'genotoul', 'genouest', 'gis', 'google', 'hebbe', 'icr_davros', 'ifb_core', 'imperial', 'imperial_mb', 'jax', 'kraken', 'mpcdf', 'munin', 'oist', 'pasteur', 'phoenix', 'prince', 'sanger', 'seg_globe', 'shh', 'uct_hpc', 'uppmax', 'utd_ganymede', 'utd_sysbio', 'uzh']
+          profile: ['abims', 'awsbatch', 'aws_tower', 'bi','bigpurple', 'binac', 'biohpc_gen', 'cbe', 'ccga_dx', 'ccga_med', 'cfc', 'cfc_dev', 'crick', 'denbi_qbic', 'ebc', 'eddie', 'eva', 'genotoul', 'genouest', 'gis', 'google', 'hebbe', 'icr_davros', 'ifb_core', 'imperial', 'imperial_mb', 'jax', 'kraken', 'mpcdf', 'munin', 'oist', 'pasteur', 'phoenix', 'prince', 'sanger', 'seg_globe', 'shh', 'uct_hpc', 'uppmax', 'utd_ganymede', 'utd_sysbio' ,'uzh']
```
Comment (ewels) [inline .github/workflows/main.yml:19]: We should really make this multi-line so the diff can be read. Fancy doing it in this PR?

**Review by ewels — APPROVED** [review]
LGTM!

File: .github/workflows/main.yml (hunk 1 above)
Comment (ggabernet) [reply .github/workflows/main.yml:19]: yes, good idea!

Comment (ggabernet) [reply .github/workflows/main.yml]: Needed to update the python cchecker.py script but now it works

**Comment by ggabernet** [comment]
I needed to update the `cchecker.py` script for that, maybe a quick review there would also be good :)

I've tested that it raises the errors properly though if the profile is not being tested.

File: docs/aws_tower.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# nf-core/configs: AWS Batch with Tower Configuration
+
+To be used when submitting jobs to AWS Batch by using Tower Forge. If you are not using Tower Forge, consider using the profile `awsbatch` where you can directly specify the Batch queue, AWS region and AWS cli path.
```
Comment (heuermh) [inline docs/aws_tower.md:3]: Perhaps add a link to Tower Forge here?

**Review by ewels — COMMENTED** [review]
oof, we were parsing the YAML as a string? Much better now - thanks! 🎉

**Pull request merged**